src/main/docker/Dockerfile-build.jvm

```diff
@@ -38,7 +38,7 @@ RUN microdnf install openssl curl ca-certificates ${JAVA_PACKAGE} \
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 
 COPY --from=build /home/jboss/target/lib/* /deployments/lib/
```

Comment:
This step will fail

Reply:
Yeah 😢 We should probably copy the entire `/target/quarkus-app` directory. I just saw it was recommended by the Quarkus team.